pages/dkp/kommander/1.3/release-notes/1.3.3/index.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Release Notes Kommander 1.3.3
+title: Release Notes Kommander 1.3.3
+menuWeight: 40
+excerpt: View release-specific information for Kommander 1.3.3
+enterprise: false
+---
+
+**D2iQ&reg; Kommander&reg; version 1.3.3 was released on x, May 2021.**
+
+[button color="purple" href="https://support.d2iq.com/s/entitlement-based-product-downloads"]Download Konvoy[/button]
+
+To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [install](/dkp/konvoy/latest/install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Known issues
+
+In versions prior to 1.3.3, Kubecost cost-analyzer by default requests a PVC of size 0.2Gi. This may cause issues with certain provisioners that require a minimum storage size of 1Gi. To resolve this issue, upgrade to 1.3.3 following the upgrade instructions below.
+
+## Upgrading
+The default cost-analyzer persistent volume size has been increased to 32Gi. Upgrades may require manual intervention if your provisioner does not support volume expansion. Here are some recommendations from the [Kubecost storage documentation](https://github.com/kubecost/docs/blob/master/storage.md):
+
+**If you are upgrading an existing version of Kommander**
```

Comment:
This info should go into an upgrade document, not the release notes.

Reply:
Where should the upgrade doc go?

Reply:
In the upgrade docs. Or maybe in the Kubecost section?